tests/api.c

```diff
@@ -13782,15 +13782,15 @@ static int test_wc_CheckProbablePrime (void)
     WC_RNG              rng;
     byte                e[3];
     word32              eSz = (word32)sizeof(e);
-    byte                n[512]; /* size of RSA_TEST_BYTES */
+    byte                n[1024]; /* size of RSA_TEST_BYTES */
```

Comment:
Might as well use a #define for this.
Should be bits / 8 for n and d. bits / 8 / 2 for p and q.

Reply:
"Might as well" --> "Criminal not to" -- another example of trying too hard to minimize the changed-octet count.  Fixed.